pkgs/by-name/in/invidious-router/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildGo122Module, fetchFromGitLab }:
+let
+  pname = "invidious-router";
+  version = "1.0";
+in
+buildGo122Module {
+  inherit pname version;
+
+  src = fetchFromGitLab {
+    owner = "gaincoder";
+    repo = "invidious-router";
+    rev = version;
+    hash = "sha256-6apw+UnhDSuPa9QB3S8b8Ej3NJTK+UCdbDjC2LkhLIg=";
+  };
+
+  vendorHash = "sha256-c03vYidm8SkoesRVQZdg/bCp9LIpdTmpXdfwInlHBKk=";
+
+  doCheck = true;
+
+  meta = {
+    homepage = "https://gitlab.com/gaincoder/invidious-router/";
```

I prefer
```suggestion
    homepage = "https://gitlab.com/gaincoder/invidious-router";
```
but that's up to you.

---

Won't change

---

Changed that